docker-compose.yml

```diff
@@ -93,6 +93,14 @@ services:
     profiles:
       - openapi
 
+  # For generating PDFs
```

Comment:
In case you didn't, it's probably a good idea to check with Koen if it's acceptable to add a new docker service.

Reply:
Yes we already spoke about this and he has reviewed the backend PR